runtime/relevance.js

```diff
@@ -40,5 +39,19 @@ class Relevance {
     // TODO: might want to make this geometric or something instead;
     return relevance / 5;
   }
+
+  static particleRelevance(relevanceList) {
+    let relevance = 1;
+    relevanceList.forEach(r => relevance *= Relevance.scaleRelevance(r));
+    return relevance;
+  }
+
+  getParticleRelevance(particleName) {
```

Comment:
This is largely subjective, but I prefer to avoid `get` prefix for methods that do non-trivial work (i.e. looping); `calc` or `find` might be better.